src/WooCommerce.php

```diff
@@ -52,6 +52,143 @@ public static function woocommerce_get_settings_gallerya(array $settings): array
     return $settings;
   }
 
+  /**
+   * Adds custom product data tabs.
+   *
+   * @implements woocommerce_product_data_tabs
+   */
+  public static function woocommerce_product_data_tabs($tabs) {
+
+    // Adds a 'Videos' tab.
+    $tabs['_' . Plugin::L10N . '_videos'] = [
+      'label'    => __('Videos', Plugin::L10N),
+      'target'   => '_' . Plugin::L10N . '_videos_product_data',
+      'class'    => ['show_if_simple', 'show_if_variable'],
+      'priority' => 21,
+    ];
+
+    return $tabs;
+  }
+
+  /**
+   * Adds custom product data panels.
+   *
+   * @implements woocommerce_product_data_panels
+   */
+  public static function woocommerce_product_data_panels() {
+
+    // Adds the tab content for 'Videos'.
+    echo '<div id="_' . Plugin::L10N . '_videos_product_data" class="panel woocommerce_options_panel hidden">';
+    woocommerce_wp_text_input([
+      'id'                => '_' . Plugin::L10N . '_video_id',
+      'value'             => get_post_meta(get_the_ID(), '_' . Plugin::L10N . '_video_id', true),
+      'label'             => __('Video ID', Plugin::L10N),
+      'description'       => __('ID of video for Vimeo or YouTube', Plugin::L10N),
+    ]);
+    woocommerce_wp_select([
+      'id'          => '_' . Plugin::L10N . '_video_source',
+      'value'       => get_post_meta(get_the_ID(), '_' . Plugin::L10N . '_video_source', true),
+      'label'       => __('Video source', Plugin::L10N),
+      'options'     => array( '' => __('Please select', Plugin::L10N), 'vimeo' => 'Vimeo', 'youtube' => 'YouTube'),
+    ]);
```

Comment:
We don't do this kind of alignment. If you add a new entry, you may need to fix alignments and this would introduce meaningless changes in the resulting commit.